packages/shared-state-babel_links_info/files/etc/shared-state/ref_state/babel_links_info_ref

```diff
@@ -0,0 +1 @@
+""
```

Comment:
Is this an empty JSON file by default? In that case is more traditional to use an empty object `{}` instead of an empty string ;-) also we could use `.json` extension to make things more explicit, if it doesn't make things much more complex

Reply:
Yes ! that was the first attempt, but when jsonc parses the text and then transforms it back to text, it creates an empty array, not an empty json object. For me it can be a bug. .. 
```lua
JSON = require("luci.jsonc")
print(JSON.stringify(JSON.parse("{}")))        
[]
```
the [documentation](https://github.com/openwrt/luci/blob/db6234c23735c6f0909679128ae85c50cf436001/libs/luci-lib-jsonc/src/jsonc.luadoc#L33) states that "Lua tables are converted into JSON arrays if they only contain integer keys, mixed tables are turned into JSON objects with any existing numeric keys converted into strings."